test/js/super.test.js

```diff
@@ -11,4 +11,23 @@ describe("super", () => {
 
   test("multiple args, with parens", () =>
     expect("super(1, 2)").toMatchFormat());
+
+  describe("with comment", () => {
```

Comment:
They all passed! 💚I also opted for a nested structure to group the 'with comment' changes:

<details>
<summary>screenshot</summary>

![Screen Shot 2020-04-24 at 9 43 14 AM](https://user-images.githubusercontent.com/4016985/80221394-3763a800-8613-11ea-932b-d44a0810cf71.png)
</details>